src/common/runtime/standalone.ts

```diff
@@ -83,7 +112,23 @@ function makeCaseHTML(t: TestTreeLeaf): VisualizedSubtree {
       await t.run(rec);
     }
 
+    result.total++;
+    result.timems = caseResult.timems;
```

Comment:
Is this supposed to be `+=`?

Reply:
Technically it doesn't matter in this case because we're starting from zero and it's only measuring one test, but I made it `+=` for consistency.